e2e/data/smoke/rook_api_external.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v1
```

Comment:
if we could replace values at runtime, it would raise more confidence that the duplication won't cause us to diverge in the future. This is a bigger discussion for another ticket though.